test/sqllogictest/types.slt

```diff
@@ -674,13 +674,23 @@ time
 query T
 SELECT format_type(1114, NULL)
 ----
-timestamp
+timestamp without time zone
 
 query T
 SELECT format_type(1184, NULL)
 ----
 timestamp with time zone
 
+query T
+SELECT format_type(1114, 3)
```

Comment:
needs tests for "invalid" values, e.g. -9, 9

Reply:
Added tests for format_type with invalid numbers. I am assuming even if the precision is invalid, it does not fail while calling `format_type`?

Reply:
Yeah, should follow PostgreSQL exactly here. Which has super weird behavior, fwiw. Invalid _negative_ numbers are discarded, but invalid positive numbers are blindly jammed in:

```
benesch=# SELECT format_type(1114, -9);
         format_type         
-----------------------------
 timestamp without time zone
(1 row)

benesch=# SELECT format_type(1114, 9);
          format_type           
--------------------------------
 timestamp(9) without time zone
(1 row)
```

I _think_ I got this logic right in the implementation, but definitely wouldn't hurt to add a test.

Reply:
Yup, the logic is the same.